src/vstest.console/Internal/ProgressIndicator.cs

```diff
@@ -115,4 +116,10 @@ private void Timer_Elapsed(object sender, System.Timers.ElapsedEventArgs e)
             }
         }
     }
+
+    public void Dispose()
+    {
+        _timer?.Dispose();
+        _timer = null;
```

Comment:
```suggestion
```

Reply:
Arg another 3h of waiting for CI.... Thanks for the suggestion!